frontend/src/routes/_dashboard.tsx

```diff
@@ -104,7 +111,7 @@ function Header() {
           <div className="flex w-full justify-center items-center">
             <Search className="relative left-10" />
             <Input
-              className="px-14 my-1 text-sm focus-visible:right-0"
+              className="px-14 my-1 placeholder:text-gray-400 text-sm focus-visible:right-0"
```

Comment:
Could you instead put this class in the `<Input>` component directly ? `placeholder:text-gray-400`

Reply:
yup sure i did that